sdk/ml/azure-ai-ml/tests/test_configs/dsl_pipeline/data_transfer_job_in_pipeline/import_file_system/pipeline.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from azure.ai.ml import Output, dsl
+from azure.ai.ml.data_transfer import import_data
+from azure.ai.ml.constants._common import AssetTypes
+from azure.ai.ml.constants._component import DataTransferTaskType
+from azure.ai.ml.entities import PipelineJob
+
+
+parent_dir = str(Path(__file__).parent)
+
+
+def generate_dsl_pipeline_from_builder() -> PipelineJob:
+    path_source_s3 = 's3://my_bucket/my_folder'
+    connection_target = 'azureml:my_s3_connection'
+    outputs = {"sink": Output(type=AssetTypes.URI_FOLDER, path="azureml://datastores/managed/paths/some_path")}
+    source = {'type': 'file_system', 'connection': connection_target, 'path': path_source_s3}
+
+    # DataTransferImport from import_data() function
+    data_transfer_function = import_data(
+        source=source,
+        outputs=outputs,
+        task=DataTransferTaskType.IMPORT_DATA,
+    )
+
+    @dsl.pipeline(description='submit a pipeline with data transfer import file system job')
+    def data_transfer_import_file_system_pipeline_from_builder(path_source_s3, connection_target):
+        from azure.ai.ml.data_transfer import FileSystem
+        s3_blob_input = data_transfer_function(source=FileSystem(**source))
```

Comment:
if we call import_data() inside @pipeline , will it be auto collected?

Reply:
does auto collected mean we can just call import_data like ```import_data_snowflake_blob = import_data(source=source_snowflake, outputs={'sink'=Output(type='mltable')})``` inside pipeline?

According to discussion before, import/export_data will behave like other component type builder function.

Reply:
offline sync with PM. It seems typo in spec. We will align import_data()/export_data() usage with command()